### DISCUSSION TITLE
How to remove hash suffix from generated files in Vite?

### DISCUSSION THREAD

**Discussion by ivanjaros** [quasarframework/quasar#14335] [General - Components / Directives / etc] [answered]
In webpack version I used this in the quasar configuration to get rid of the hash suffixes for generated files:
```
chainWebpack (chain) {
        chain.plugin('eslint-webpack-plugin')
          .use(ESLintPlugin, [{ extensions: [ 'js', 'vue' ] }])
      },
```

in Vite I am trying variations of:
```
extendViteConf (viteConf) {
        Object.assign(viteConf.build, {
          rollupOptions: {
            output: {
              entryFileNames: "[name].js"
            }
          }
        })
        // viteConf.build.rollupOptions = {
        //   output: {
        //     entryFileNames: "[name].js"
        //   }
        // }
      },
```

but it has no effect.

**Comment by rstoenescu** [comment, marked as answer]
Hi. This is a Vite specific configuration, so edit quasar.config.js > build:

```
extendViteConf (viteConf) {
        viteConf.build.rollupOptions = viteConf.build.rollupOptions || {}
        viteConf.build.rollupOptions.output = viteConf.build.rollupOptions.output || {}

        const target = viteConf.build.rollupOptions.output
        const assetsDir = (viteConf.build.assetsDir || 'assets') + '/'

        target.entryFileNames = `${ assetsDir }/[name].js`
        target.chunkFileNames = `${ assetsDir }/[name].js`
        target.assetFileNames = `${ assetsDir }/[name].[ext]`
      }
```

However, this will also come natively in q/app-vite in v1.1 through: quasar.config.js > build > useFilenameHashes = false, but if you make the changes above you don't need to wait for it.

**Comment by ivanjaros** [comment]
seems to be working except for web fonts(this is default instance with ZERO changes):
![obrázok](https://user-images.githubusercontent.com/64546059/189700565-3166a205-0ef3-4516-b4fc-e23f9b14775d.png)

**Reply by rstoenescu** [reply]
> seems to be working except for web fonts(this is default instance with ZERO changes):
> 
> ![obrázok](https://user-images.githubusercontent.com/64546059/189700565-3166a205-0ef3-4516-b4fc-e23f9b14775d.png)
> 
> 
> 
> 

Not really. Those are the actual original filenames for fonts.
